CHANGELOG.md

```diff
@@ -1 +1,19 @@
+# Change Log
 
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+The format is based on [Keep a Changelog](http://keepachangelog.com/).
+
+## Version 1.0.0
+
+### Added
+
+Initial release that provides the following features 
+
+- Create attachment : Provides the capability to upload new attachments.
+- Open attachment : Provides the capability to preview attachments.
+- Delete attachment : Provides the capability to remove attachments.
+- Rename attachment : Provides the capability to rename attachments.
+- Virus scanning : Provides the capability to support virus scan for virus scan enabled repositories.
+- Draft functionality : Provides the capability of working with draft attachments.
+- Repository-specific attachment display: Enables viewing attachments available in the repository linked to the deployed application.
```

Comment:
Display attachments specific to repository: Lists attachments contained in the repository that is configured with the CAP application.

Reply:
Changed